site/content/faq/how-can-i-run-zap-with-a-high-dpi-display.md

```diff
@@ -9,30 +9,21 @@ weight: 7
 If ZAP is displayed in a really tiny window then it's probably because you have
 a high DPI display.
 
-The solution is easy - use Java 9+
```

Comment:
Change to say that this should not be a problem on Windows now with Java 11+ ?

Reply:
The following advice also no longer seems to be needed.

Reply:
The bit about the bat? People may still have multiple versions and need to be specific.

Reply:
Yes, but that would be unrelated to HiDPI now, you need the newer version whether the UI is properly displayed or not. Maybe move to other FAQ entry.

Reply:
Okay